content/en/tracing/setup_overview/setup/java.md

```diff
@@ -241,7 +241,7 @@ System properties can be set as JVM flags.
 | `dd.trace.agent.timeout`               | `DD_TRACE_AGENT_TIMEOUT`               | `10`                              | Timeout in seconds for network interactions with the Datadog Agent.                                                                                                                                                                                                   |
 | `dd.trace.header.tags`                 | `DD_TRACE_HEADER_TAGS`                 | `null`                            | (Example: `CASE-insensitive-Header:my-tag-name,User-ID:userId`) A map of header keys to tag names. Automatically apply header values as tags on traces.                                                                                                               |
 | `dd.trace.annotations`                 | `DD_TRACE_ANNOTATIONS`                 | ([listed here][8])               | (Example: `com.some.Trace;io.other.Trace`) A list of method annotations to treat as `@Trace`.                                            |
-| `dd.trace.methods`                     | `DD_TRACE_METHODS`                     | `null`                            | (Example: `"package.ClassName[method1,method2,...];AnonymousClass$1[call]"`) List of class/interface and methods to trace. Similar to adding `@Trace`, but without changing code.                                                                                       |
+| `dd.trace.methods`                     | `DD_TRACE_METHODS`                     | `null`                            | (Example: `"package.ClassName[method1,method2,...];AnonymousClass$1[call];package.ClassName[*]"`) List of class/interface and methods to trace. Similar to adding `@Trace`, but without changing code. Note: wildcard method support does not accommodate Constructors, Getters, Setters, Synthetic, toString, Equals, HashCode and Finalizer method calls                                                                                       |
```

Comment:
```suggestion
| `dd.trace.methods`                     | `DD_TRACE_METHODS`                     | `null`                            | (Example: `"package.ClassName[method1,method2,...];AnonymousClass$1[call];package.ClassName[*]"`) List of class/interface and methods to trace. Similar to adding `@Trace`, but without changing code. **Note:** The wildcard method support (`[*]`) does not accommodate constructors, getters, setters, synthetic, toString, equals, hashcode, or finalizer method calls                                                                                       |
```
If lowercasing the method call descriptions changes the technical meaning, you can ignore that part of my edit, but they seemed like regular words to me, not proper nouns.

Reply:
These are proper nouns, however I think developers will clearly understand the expectations here eitherway